ArjunaCore/arjuna/pom.xml

```diff
@@ -94,7 +94,7 @@
       <groupId>io.mashona</groupId>
       <artifactId>mashona-logwriting</artifactId>
       <version>1.0.0.Beta1</version>
-      <scope>provided</scope>
+      <scope>compile</scope>
```

Comment:
Is it intentional to also add this to the runtime classpath (which is what would happen by making compile)? https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html

Reply:
Jonathan originally had it as provided which I understood to mean that it was provided by the JDK so it was confusing why it only failed on JDK 11 (presumably it is a modules issue). But as a minimum it is needed for compilation. Let's wait for Jonathan to return.

Reply:
In maven terms provided is when the dependency is available present within the runtime environment already, so yeah if the JDK is providing it it seems that it should not be added in by Narayana too. But if the JDK is providing it then also I would think that it wouldn't have to be added for compilation (e.g. we don't need to for java.lang). as you say, it could be a modules thing we need to add. That said, would the JDK have io.mashona:mashona-logwriting things in it?

Reply:
I think I was on a bit the wrong steer as provided is also present at compile too. So this is about exposing it to the javadoc compiler which might help explain the other change